src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -307,21 +307,22 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
             // if we found an Account already connected to this Contact, connect the Contact to that Account if it
             // is the correct type for the current account processor
             Account acc = mapContactIdAccount.get(c.Id);
-            if (acc != null && (acc.RecordTypeId == UTIL_Describe.getHhAccRecTypeID() 
-            || acc.RecordTypeId == UTIL_Describe.getAdminAccRecTypeID())) {
+            if (acc != null && (acc.RecordTypeId == UTIL_CustomSettingsFacade.getSettings().Account_Processor__c)) {
                 UTIL_Debug.debug('****Account already existed');
                 // if a user has blanked out the Account for a Contact, this will put it right back
                 c.AccountId = acc.Id;                
             } else {
             	UTIL_Debug.debug('****Creating new account');
                 
-                // construct the individul account for the Contact
+                // construct the individual account for the Contact
                 Account a = new Account();
                 
                 // construct the Account name
-                if(defaultRecTypeID == UTIL_Describe.getHhAccRecTypeID())
+                if(defaultRecTypeID == UTIL_CustomSettingsFacade.getSettings().Household_Addresses_RecType__c)
```

Comment:
why are you looking at an Address related record type vs an Account record type?  or am I just misunderstanding what your setting is holding?